{"name": "实名认证", "status": "broken", "statusDetails": {"message": "AttributeError: 'tuple' object has no attribute 'url'", "trace": "token = 'eyJ0aW1lT3V0IjoxNjAyNjg2NTUyMDkxLCJ1c2VySWQiOjEzMDYsInVzZXJOYW1lIjoiMTIzempjIn0='\n\n    @allure.title(\"实名认证\")\n    def test_realname(token):\n        headers = {\n            'token': token,\n            'charset': 'UTF-8'\n        }\n        url = 'http://qa.yansl.com:8084/cst/realname2'\n        email = random_tool.random_email()\n        sex = random_tool.random_sex()  # 随机生成男(1)或女(0)\n        cst_name = random_tool.random_name(sex)\n        area = random_tool.random_area()\n        area_id = area['code']\n        area_name = area['name']\n        province = area['province']\n        city = area['city']\n        birth_days = random_tool.random_birthday(18, 35)\n        cert_no = identity_tool.generate_id(sex=sex, area_id=area_id, birth_days=birth_days)\n        req ={\n                  \"cstId\": 15790,\n                  \"customerInfo\": {\n                       \"birthday\": '2020-10-31',\n                       \"certno\": cert_no,\n                       \"city\":  city,\n                       \"cstName\": cst_name,\n                       \"email\":email,\n                       \"province\": province,\n                       \"sex\": sex\n                  }\n                   }\n        print(req)\n>       resp = request_tool.post_json(url=url,headers=headers, json=req)\n\ntest_case\\User\\test_realname.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ()\nkwargs = {'headers': {'charset': 'UTF-8', 'content-type': 'application/json;;charset=UTF-8', 'token': 'eyJ0aW1lT3V0IjoxNjAyNjg2...'certno': '23032119961014897X', 'city': '鸡西市', 'cstName': '邵广', ...}}, 'url': 'http://qa.yansl.com:8084/cst/realname2'}\nr = ()\n\n    def _func(*args, **kwargs):\n        r= func(*args, **kwargs)\n        request = \"-------------------request-------------\" \\\n>                 \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\nE       AttributeError: 'tuple' object has no attribute 'url'\n\ntools\\report\\decorators_tool.py:11: AttributeError"}, "attachments": [{"name": "log", "source": "e2ec46c5-2107-4546-8ce9-ef82ef7711ef-attachment.txt", "type": "text/plain"}], "start": 1602684760074, "stop": 1602684771635, "uuid": "6b903987-c5ff-4d47-849b-62f71b8d8103", "historyId": "d6666fcad673fa0ef5e5af13237f96e1", "fullName": "test_case.User.test_realname#test_realname", "labels": [{"name": "parentSuite", "value": "test_case.User"}, {"name": "suite", "value": "test_realname"}, {"name": "subSuite"}, {"name": "host", "value": "DESKTOP-679Q8H2"}, {"name": "thread", "value": "17252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.User.test_realname"}]}